config/ingestion.example.yml

```diff
@@ -1,3 +1,43 @@
 name: production
 logfile: log/ingestion.log
+loglevel: 0
```

Comment:
which loglevels exist, and is there a friendly name for the level you have selected now?